books/RayTracingInOneWeekend.html

```diff
@@ -2224,6 +2224,8 @@
         }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     };
+
+    ...
```

Comment:
We generally understand an implicit leading `...` and trailing `...` for listings when changes are scoped to an existing function or class.